app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_people = []
+
+    for person in people:
+        list_of_people.append(Person(person["name"], person["age"]))
+
+    for person in people:
+
+        if "wife" in person and person["wife"]:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+
+        elif "husband" in person and person["husband"]:
+            Person.people[person["name"]].husband = (
+                Person.people)[person["husband"]]
```

Comment:
bad formmating here - no need to use brackets for breaking line here, could u break the line by existing ones ?
also the braces must be located in one of two options: open and start with the text or have a line break between the text